src/components/GameCard/GameCard.module.scss

```diff
@@ -123,6 +123,10 @@ $hoverBorderThickness: 2px;
   @include visibleElement();
   transition: 0.1s;
   position: relative;
+
+  @include for-mobile-only {
+    font-size: var(--text-md) !important;
```

Comment:
Needed to add !important here because the title-mixin from the topography is overriding this variable and always setting the title to --text-xl.

Since I don't want to mess with the typography I think its better to add important here.

Reply:
we usually try to stick to the typography types. Title already resizes a bit at some breakpoints is that not enough?

Reply:
Yeah, I don't think the breakpoints are working for the gamecard titles (from the gameinfov2 :
![image](https://github.com/user-attachments/assets/eec8c8d5-d6f0-43e1-999e-ca122045b527)

I discussed this with david here:
https://github.com/HyperPlay-Gaming/px-pm/issues/238